src/App.vue

```diff
@@ -44,17 +43,28 @@ export default {
       try {
         let url = updateQueryStringParameter(`${process.env.VUE_APP_BASE_URL}/ws-server`, 'username', username);
         url = updateQueryStringParameter(url, 'lobby', lobby);
-        this.connection = new HubConnectionBuilder().withUrl(url).build();
+
+        if (localStorage.id) {
+          const sessionId = localStorage.id;
+          url = updateQueryStringParameter(url, 'sessionId', sessionId);
+          this.id = sessionId;
+          localStorage.id = sessionId;
```

Comment:
You can remove this line. sessionId was instantiated already with the value of localStorage.id

Reply:
removed